modules/retail/pages/retail-deploy-terminals.adoc

```diff
@@ -135,3 +135,36 @@ When you have your bootstrap image ready, you can deploy the image to the termin
 
 . Power on your POS terminals.
 . The branch server will bootstrap the terminals according to the data you have provided.
+
+
+
+== Re-Deploy Images to the Terminals
+
+You can instruct terminals to download and deploy images every time they are restarted.
+This is achieved using a Salt state.
+
+[[Warning]]
+====
+Re-deploying a terminal removes all data stored on the terminal hard disk, including any persistent partitions.
```

Comment:
Only terminal repartition destroys all data. Image redeployment is usually safe. Can you move this to the second procedure?